dune_daq_services/variables/dunedaq.yaml

```diff
@@ -14,5 +14,5 @@ dunedaq:
     protocol_version: "3.6"
     replicas: 1
   unified_microservices:
-    image: ghcr.io/dune-daq/microservices
-    tag: fa6c
+    image: ghcr.io/dune-daq/pocket-runserv-rest
```

Comment:
I'd prefer to get the feature merged into microservices and pull from there.  This variable change impacts all microservice images.

Reply:
Completely understandable, I wasn't expecting to merge this into develop until we had the unified microservices image built and posted